bindings/wasm/docs/api-reference.md

```diff
@@ -2036,7 +2035,7 @@ on the same document. This is intended for signing updates to a document where a
 capability invocation method is rotated or replaced entirely.
 
 NOTE: does not validate whether the private key of the given `key_pair` corresponds to the
-verification method. See [Document.verifyDocument](Document.verifyDocument).
+verification method. See [#Document+verifyDocument](#Document+verifyDocument).
```

Comment:
We generally just remove intra-doc links in the Wasm bindings so I would be fine just rendering this as code without a link.

Otherwise maybe just changing the text to match the function call would be better.

![image](https://user-images.githubusercontent.com/5772887/178038517-e70e960f-ebda-4839-b874-d99e65034106.png)

```suggestion
verification method. See [Document.verifyDocument](#Document+verifyDocument).
```